src/extension.ts

```diff
@@ -31,6 +31,65 @@ export function activate(context: vscode.ExtensionContext) {
 	});
 	context.subscriptions.push(disposable_number_cursors_from_one);
 	
+	let disposable_number_cursors_from_arbitrary = vscode
+			.commands
+			.registerTextEditorCommand('engineernick.multi-cursor-tools.number_cursors_from_arbitrary', (editor, edit, args) => {
+		vscode.window.showInputBox({
+			placeHolder: "1",
+			prompt: "Starting number",
+			value: 1
```

Comment:
Object must match the interface `vscode.InputBoxOptions` which requires that `value` is a string;

```suggestion
			value: "1"
```

Reply:
Thanks for that ... sorry for getting the wrong type